### PR TITLE
Add Enqueue job queue pack

### PR DESCRIPTION
**Pull request by makasim**
| Q             | A
| ------------- | ---
| License       | MIT

A pack to install Enqueue's job queue with Doctrine ORM as a storage in a Symfony Flex application

### CODE REVIEW

**Comment by makasim** [comment]
@symfony-flex-server please review.

**Comment by fabpot** [comment]
@symfony-flex-server review please (@makasim that's the right way :)).

**Comment by makasim** [comment]
@fabpot I still miss something. Why hasn't it been merged?

File: enqueue/job-queue-pack/0.8/manifest.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/"
```
Comment (msheakoski) [inline enqueue/job-queue-pack/0.8/manifest.json]: This should be `"config/": "%CONFIG_DIR%/"` and the `etc/packages` dir in the recipe needs to be renamed to `config/packages`

**Pull request merged**